### PR TITLE
MassInstallAction: Install the CI workflow on this repository

### PR DESCRIPTION
**Pull request by timholy**
This pull request sets up the CI workflow on this repository.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1'
+#          - 'nightly'
```
Comment (johnnychen94) [inline .github/workflows/CI.yml]: is this deliberate?

**Review by johnnychen94 — COMMENTED** [review]
we also need to set up the documentation here.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (timholy) [reply .github/workflows/CI.yml]: Some people complained about GitHub Actions not providing the equivalent `allow-failures`.

**Comment by timholy** [comment]
#44. I also added the necessary keys.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (johnnychen94) [reply .github/workflows/CI.yml]: I see. The current workaround to this is to disable `fail-fast` and manually ignore them 😆

The whole Images stack is friendly to nightly Julia builds so I guess we should keep this good practice?

Comment (timholy) [reply .github/workflows/CI.yml]: I'm happy to add it back. Will get to it in a bit.

**Review by johnnychen94 — APPROVED** [review]
Eventually, we may want to set up an organizational GA template and share it among all the repos under JuliaImages. We can also set up an org level secret (I've already set up one `TAGBOT`).

Ref: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization

**Comment by timholy** [comment]
The test failure on Julia 1.0 has the following explanation:
- ImageContrastAdjustment's dependencies allow Reexport 1.x to be installed
- at testing time, it needs to additionally install FFTW
- the only version of FFTW supporting Reexport 1.x is FFTW 1.3 or higher
- no FFTW version after 1.1 supports Julia 1.0

This could all be addressed by installing an earlier release of Reexport, but Julia 1.0 is not capable of downgrading packages in order to satisfy test dependencies.

How do folks want to handle this? I propose we ignore the test failure, but we could also stop supporting Julia 1.0. Finally, we could submit a 1.1.1 release of FFTW supporting Reexport 1.0. Maybe that's the best option?

EDIT: xref https://github.com/JuliaMath/FFTW.jl/commit/8d35ae8fd1819108ae1c6995031a2f5ba6156454. :crossed_fingers:  that this fixes it.

**Comment by timholy** [comment]
Sadly, that was not enough to fix it. AbstractFFTs gets installed at v1.0, and that's only supported by FFTW 1.3+.

I propose we ignore the failure on Julia 1.0.

**Comment by timholy** [comment]
Since this has nothing to do with this PR, I'm merging.

**Pull request merged**